cap/csp/cache.go

```diff
@@ -0,0 +1,63 @@
+package csp
+
+import (
+	"context"
+	"crypto/md5"
+
+	api "github.com/wetware/pkg/api/process"
+)
+
+// HashSize is the size of the hash produced by HashFunc.
+const HashSize = md5.Size
+
+// HashFunc is the function used for hashing in the default
+// executor implementation.
+// TODO switch to more suitable hashing function, e.g. BLAKE3.
+var HashFunc func([]byte) [HashSize]byte = md5.Sum
```

Comment:
I would recommend using `ww.ROM.CID()` to uniquely identify the WASM code.  It uses a `BLAKE3` multihash, and is what we'll later be using to fetch ROMs over bitswap.  In the meantime, reusing that lets us remove code from this PR.

Reply:
Using `ww.ROM` from this package creates a dependency cycle, what do you think about moving it to a separate package, like the existing `csp`?